### PR TITLE
Remove Trackr integration (API removed) - Updated

### PR DESCRIPTION
**Pull request by FuzzyMistborn**
## Breaking change

The Trackr integration should be removed from HomeAssistant as the API is now gone and the Android/iOS devices are no longer supported.

Screwed up the PR in https://github.com/home-assistant/home-assistant.io/pull/19254 so closed and reopening here.

## Proposed change

The Trackr integration is dead now that the API is killed.  The [website ](https://www.thetrackr.com/) says that the Android/iOS app is no longer supported which basically kills the point of the integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [X] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/55917
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2803
- This PR fixes or closes issue: https://github.com/home-assistant/core/issues/54765

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_redirects [hunk 1]
```diff
@@ -1893,7 +1892,6 @@
 /components/tplink /integrations/tplink
 /components/tplink_lte /integrations/tplink_lte
 /components/traccar /integrations/traccar
-/components/trackr /integrations/trackr
```
Comment (MartinHjelmare) [inline source/_redirects:1896]: Please add a redirect under removed integrations.

https://github.com/home-assistant/home-assistant.io/blob/8b846e5bb1489ad576d050aca788397a02255c0c/source/_redirects#L2210

Comment (FuzzyMistborn) [reply source/_redirects:1896]: Ahh missed that.  Done.

File: source/_redirects [hunk 2]
```diff
@@ -2314,3 +2312,5 @@
 /components/wunderground /more-info/removed-integration 301
 /integrations/wunderground /more-info/removed-integration 301
 /integrations/garmin_connect /more-info/removed-integration 301
+/components/device_tracker.trackr /more-info/removed-integration 301
+/components/trackr /more-info/removed-integration 301
```
Comment (MartinHjelmare) [inline source/_redirects:2316]: Please add a redirect for `/integrations/trackr` too.

Comment (FuzzyMistborn) [reply source/_redirects:2316]: Oops.  Need more ☕

File: source/_redirects [hunk 3]
```diff
@@ -2314,3 +2312,6 @@
...
+/integrations/trackr/ /more-info/removed-integration 301
```
Comment (MartinHjelmare) [inline source/_redirects]:
```suggestion
/integrations/trackr /more-info/removed-integration 301
```

**Review by FuzzyMistborn — COMMENTED** [review]
Thanks.

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**